compiler/luci-interpreter/src/loader/KernelBuilder.test.cpp

```diff
@@ -532,42 +533,63 @@ TEST_F(KernelBuilderTest, Prelu)
   checkTensor(kernel->output(), op);
 }
 
-TEST_F(KernelBuilderTest, ResizeNearestNeighbor)
```

Comment:
Sorry for confusing. I found that this TC is not located at correct position(on alphabetical order). so I make a change on this.